boa/src/syntax/ast/node/mod.rs

```diff
@@ -116,6 +118,9 @@ pub enum Node {
     /// An 'if' statement. [More information](./conditional/struct.If.html).
     If(If),
 
+    /// A labelled Statement.
+    Label(Label),
```

Comment:
Here too, I would use the same name as in the spec: `LabelledStatement`. In fact, I think we should change other node variant names too, but in a different PR.